samples/besu/simpleasset/contracts/manageAssetAny.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.8;
+
+import "./transferInterface.sol";
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC1155/utils/ERC1155Holder.sol";
+
+/**
+ * Hashed Timelock Contract on Fungible Assets in an Ethereum network (Support only for ERC20 tokens right now)
+ **/
+
+contract InteroperationBaseClassERCAny is ERC1155Holder {
+    mapping(bytes32 => LockContract) public lockContracts;
+
+    struct LockContract {
+        address sender;
+        address receiver;
+        address assetContract;
+        uint256 amount;
+        bytes32 hashLock;
+        uint256 expirationTime;
+        uint8 status;
+        uint256 tokenId;
+        bytes data;
+    }
+
+    struct transferInfo {
+        address sender;
+        address receiver;
+        uint256 amount;
+    }
+
+    uint8 constant UNUSED = 0;
+    uint8 constant LOCKED = 1;
+
+    event Lock(
+        address indexed sender,
+        address indexed receiver,
+        address assetContract,
+        uint256 amount,
+        bytes32 hashLock,
+        uint256 expirationTime,
+        bytes32 lockContractId
+    );
+
+    event Claim(
+        address indexed sender,
+        address indexed receiver,
+        bytes32 indexed lockContractId,
+        bytes32 hashLock,
+        bytes32 preimage
+    );
+
+    event Unlock(
+        address indexed sender,
+        address indexed receiver,
+        bytes32 indexed lockContractId
+    );
+
+    // The sender locks a fungible asset with a hash lock and an expiry time
+    function lockFungibleAsset(
```

Comment:
These functions are for any i.e. fungible or non-fungible or hybrid assets right? Can we remove `fungible` from the names of all the functions?